### ISSUE TITLE
Feature request: Map Alt-F2 to Meta (Windows) key

### ISSUE THREAD

**Issue by probonopd** [rodlie/draco#34]
Map Alt-F2 to Meta (Windows) key so that users coming from other platforms/desktop environments can find it more easily. Especially as long as pressing that key (alone) does nothing otherwise like open the applications menu.

![](https://user-images.githubusercontent.com/2480569/81451610-6847ef00-9174-11ea-89f9-28a624f243d6.png)

_Screenshot showing AppImages on draco, integrated using `appimaged` daemon from [go-appimage](https://github.com/probonopd/go-appimage/tree/master/src/appimaged)_

**Comment by rodlie** [comment]
Yeah, I'm used to "the old way" from Fluxbox, Openbox etc. Currently this is hardcoded, but should at least be a setting.

**Comment by rodlie** [comment]
https://github.com/rodlie/draco/commit/05c99a8d3e216bb2c0fbffeeb089d284f5882ca4

You can now override alt+f1 and alt+f2 in ~./.config/draco/sessionsettings.conf
```
[shortcuts]
terminal="FOO"
launcher="FOO"
```

Note that Openbox uses ``win`` for various stuff, see:
https://github.com/rodlie/draco/blob/master/share/theme/rc.xml, override in ~/.config/draco/rc.xml

**Comment by probonopd** [comment]
Thanks @rodlie. What do I need to enter there to get Alt-F2 remapped to Windows+<nothing> and/or Alt-Spacebar?
